### ISSUE TITLE
Идея: Взять механ для фонариков с Эриса

### ISSUE THREAD

**Issue by SierraHelper** [SierraBay/SierraBay12#1462]
**Название:**
`Куда смотришь - туда и свет.`

**Дополнительная информация:**
На Эрисе тоже механика фонарика светить перед тобой, однако, когда ты нажимаешь Осмотреть на любой тайл - фонарик магическим образом светит туда! Вауууу! Считаю это круто.
<hr>

*Репорт сгенерирован автоматически*
*Автор: `honkemo` / `598615241185689661`*

**Comment by SierraHelper** [comment]
### [Обсуждение в Discord](https://discord.com/channels/617003227182792704/1173750641508761694)

**Comment by SuhEugene** [comment]
Да, это круто. Но запросы - для любых репортов, которые касаются мелких шероховатостей, а не новая предложка. Так что ишью отклонён.

**Comment by SuhEugene** [comment]
Открываю как идею

**Comment by LordNest** [comment]
Все запросы ранее 2024 года закрываю по причине их устаревания. Серьёзно, если мы что-то не ввели за два года, то это ну, скорее всего не будет введено без повторного пинка.
